src/blocks/BlockArg.as

```diff
@@ -94,6 +94,7 @@ public class BlockArg extends Sprite {
 			base = new BlockShape(BlockShape.RectShape, c);
 			this.menuName = menuName;
 			addEventListener(MouseEvent.MOUSE_DOWN, invokeMenu);
+			editable = true; // because why not
```

Comment:
So from what I'm guessing you can type things in menu inputs?

That's cool! Especially for tabbing through inputs.

I wonder what the point of darkening some menus but not others is, then..